### ISSUE TITLE
Retire Echo Nest plugin in favor of new Spotify API

### ISSUE THREAD

**Issue by sampsyo** [beetbox/beets#1920]
The Echo Nest API is shutting down. From an email I got today:

> 1. March 29th, 2016 - As of March 29, 2016, we are announcing three new APIs (details below) and The Echo Nest API will no longer be issuing new API keys
> 2. May 31, 2016 - As of May 31, 2016 The Echo Nest platform will no longer serve requests and developers will need to move over to the Spotify API.

The news could be worse; they've simultaneously [added equivalent features to the Spotify API](https://developer.spotify.com/news-stories/2016/03/29/audio-features-recommendations-user-taste/). But we need to migrate, and it seems that the new endpoint requires authentication with a per-user Spotify account.

**Comment by pprkut** [comment]
The devil lies in the detail unfortunately :( "Our new Audio Features API makes all of this analysis available for every track in Spotify’s catalog."
If I'm reading this correctly the new API will only work for titles in Spotify's catalog, no more option to upload and analyze. The fact that the web service to get the audio features data requires a spotify ID seems to confirm that.

**Comment by sampsyo** [comment]
Ugh, yes, that is bad news. :unamused:

**Comment by bearcatsandor** [comment]
So why not drop echonest and put our efforts behind integration with http://acousticbrainz.org/ instead, since it hooks into our musicbrainz backed?

**Comment by sampsyo** [comment]
Yep, we do have an AB plugin already, which makes this much less urgent.
